scripts/exportexcel.py

```
@@ -0,0 +1,97 @@
+import pandas as pd
+import sys
+from openpyxl.utils import get_column_letter
+
+# Get the current working directory
+current_dir = os.getcwd()
+
+# Add it to sys.path
+if current_dir not in sys.path:
+    sys.path.append(current_dir)
+
+from src.jsonsync import JsonItem
```

Ruff (0.12.2)

Module level import not at top of file

(E402)